DataSetDocker/ReadMe.md

```diff
@@ -35,20 +35,19 @@ docker run -p $(docker-machine ip $(docker-machine active)):8888:8888 dataset
 
 ## For Linux:
 
-`ip = 127.0.0.1`
+```
+ip = 127.0.0.1
+```
 
 ## For windows:
 
-`ip = docker-machine ip $(docker-machine active)`
-
-# To access the jupyter notebook:
-
-Now the console with jupyter-lab server is running.
+```
+ip = docker-machine ip $(docker-machine active)
+```
 
-1. Copy the address with token from the console into your browser:
+# To access the jupyter notebook in your browser type:
 
 ``` 
-<ip>:8888/?token=1708633ec3f24c945c8cc18ce0ca40f90d2ac94a0a55119f
+<ip>:8888
 ```
 
```

Comment:
Could you add a comment here about what happens when this is launched? i.e. that there is a note book that can be used to generate all examples and any other details.